.github/workflows/containers.yml

```diff
@@ -8,13 +8,22 @@ jobs:
       matrix:
         gtk-version: [gtk3, gtk4]
         linux-distro: [debian, fedora, opensuse, ubuntu]
-
-    name: ${{ matrix.linux-distro }} ${{ matrix.gtk-version }}
+        arch: [amd64]
+        include:
+          - gtk-version: gtk3
+            linux-distro: debian
+            arch: 386
```

Comment:
`i386` would be more idiomatic, would not be interpreted as an `int` by accident and also is in line with Debian's architecture naming (`amd64` is, too), which might make some things more straightforward.

Reply:
`386` is the arch of the image: https://hub.docker.com/layers/library/debian/buster/images/sha256-abea34421f483e6dfe65a4f7d5267124a5de9b53e522c27bc9eb935644229ecb?context=explore
We cannot change it.

Reply:
I see. I've always used the `i386/debian` image. These prefixes are clearly inspired by Debian. I don't mind using `386` then. Maybe it should be put in quotes, though, to make sure it's considered a string.

Reply:
Well, it seems `i386` is also valid and pulls the correct image. Let's see CI result with bba401749cad29b04d11f37d8ebcb8baf82b2429 then.

Reply:
```
Run buildah bud --arch="${ARCHITECTURE}" --build-arg MACHINE="${MACHINE}" --tag "${IMAGE_NAME}" --file "containers/${GTK_VERSION}/Dockerfile.${LINUX_DISTRO}" .
[1/2] STEP 1/13: FROM docker.io/debian:buster AS build-stage
Trying to pull docker.io/library/debian:buster...
error creating build container: choosing an image from manifest list docker://debian:buster: no image found in manifest list for architecture i386, variant "", OS linux
[2/2] STEP 1/7: FROM docker.io/debian:buster
time="2023-09-17T12:41:48Z" level=error msg="exit status 125"
Error: Process completed with exit code 125.
```
Nope, `i386` is not understood by Buildah.

Reply:
I really don't insist on it. Your first attempt worked fine. I was just a bit curious on why it was that value specifically.